.github/workflows/mirror_community_pipeline.yml

```diff
@@ -0,0 +1,54 @@
+name: Fast tests for PRs
+
+on:
+  # Push changes on the main branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'examples/community/**.py'
+
+  # And on tag creation
+  create:
+    tags:
+      - '*'
+jobs:
+  mirror_community_pipeline:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup + install dependencies
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install uv
+          uv pip install --upgrade huggingface_hub
+
+      # Determine to which folder to push changes (based on branch or tag)
+      - name: Infer path in repo (main or tag)
+        run: |
+          if [ "${GITHUB_EVENT_NAME}" == "push" ]; then
+            if [ "${GITHUB_REF}" == "refs/heads/main" ]; then
+              echo "PATH_IN_REPO=main" >> $GITHUB_ENV
+            else
+              echo "PATH_IN_REPO=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+            fi
+          elif [ "${GITHUB_EVENT_NAME}" == "push" ] && [ "${GITHUB_REF}"!= "refs/heads/main" ]; then
+            echo "PATH_IN_REPO=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          fi
+
+      # Check secret is set
+      - name: whoami
+        run: huggingface-cli whoami
+        env:
+            HF_TOKEN: ${{ secrets.HF_TOKEN_MIRROR_COMMUNITY_PIPELINES }}
+
+      # Push to HF!
+      - name: Mirror community pipeline to HF
+        run: huggingface-cli upload Wauplin/diffusers-community-pipelines-mirror ./examples/community ${PATH_IN_REPO} --repo-type dataset
```

Comment:
Destination dataset repo id to be updated.

Reply:
Good catch! 

Added it in https://github.com/huggingface/diffusers/pull/8417/commits/2d9a9ecfffc73236dc8c067f3d4635ac2b0148ca